File: Iniciante/URI 2786.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val L: Int = leitor.nextInt();
    val C: Int = leitor.nextInt();
    val lajes1: Int = (L * C) + ((L - 1) * (C - 1));
    val lajes2: Int = ((L - 1) * 2) + ((C - 1) * 2);
    
    println(lajes1);
    println(lajes2);
}

File: Iniciante/URI 2786.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val L = leitor.nextInt()
        val C = leitor.nextInt()
        val lajes1 = (L * C) + ((L - 1) * (C - 1))
        val lajes2 = ((L - 1) * 2) + ((C - 1) * 2)
        println(lajes1)
        println(lajes2)
    }
	
}